00_download_data/1C_preprocess_450k_methylation_data.ipynb

```diff
@@ -0,0 +1,471 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Preprocess pan-cancer 450K methylation data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Load the downloaded data and curate sample IDs.\n",
+    "\n",
+    "TODO: look at cancer type composition of represented samples (are any missing?)"
```

Comment:
There's still a TODO here, you may want to convert it to a Github issue so it doesn't get lost

Reply:
Good idea! Created #16